### PR TITLE
Removes the dance that makes you rest from the dance machine

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Removes the dance of the dancing machine that makes you rest. Thus making you drop items. Criminals and such.

**Changelog:**
:cl:
del: Removes the dance from the dancing machine that makes you lay down
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
_EVERYBODY DO THE FLOP_
_flops_

**Comment by Couls** [comment]
Nooo I love that john wick moment when security is chasing you and you run into the occasional dance party and they're forced to get down and boogie and drop their batons

File: code/game/machinery/dance_machine.dm [hunk 1]
```diff
@@ -322,14 +322,12 @@
...
-	switch(rand(0,9))
+	switch(rand(0,6))
 		if(0 to 1)
 			dance2(M)
 		if(2 to 3)
 			dance3(M)
 		if(4 to 6)
-			dance4(M)
-		if(7 to 9)
 			dance5(M)
```
Comment (Citinited) [inline code/game/machinery/dance_machine.dm]: If you're going to do this, I'd suggest renaming `dance5` to `dance4` to keep things consistent, it's probably just me though

**Comment by MrMagolor** [comment]
>Nooo I love that john wick moment when security is chasing you and you run into the occasional dance party and they're forced to get down and boogie and drop their batons

On the flipside isn't it incredibly annoying when you're an antag and it makes you drop _your_ weapon?

**Comment by taukausanake** [comment]
We should add the boogie bomb from Fortnite just because of this PR. That would be hilarious

**Comment by Spacemanspark** [comment]
Alternatively a different dance machine that doesn't make people dance at all would be nice. Especially for admin shenanigans tbh.

**Comment by Citinited** [comment]
Imagine if emagged dance machines played Disco Inferno on repeat and made everyone around it burst into flame, while spewing plasma :thinking:

Alternatively, dance machines make people bust different moves depending on how groovy the track is (and how long each individual has been jiving for)

**Comment by taukausanake** [comment]
What if you made it so that you have to be on Help intent for you to dance, and any other intent you don't? I think that would be good

**Comment by farie82** [comment]
If somebody else wants to take on the PR feel free.

**Comment by farie82** [comment]
Marsmond has a better fix I just noticed. Closing this one in favor of his.

**Pull request closed without merging**